fastlane/Fastfile

```diff
@@ -248,6 +256,56 @@ lane :download_translations do |commit_changes: false|
   end
 end
 
+# Synchronizes localization files with GlotPress and creates a PR
+#
+# This lane performs a complete localization sync:
+# 1. Generates source PO file from English Fluent file
+# 2. Downloads latest translations from GlotPress
+# 3. Creates a PR with the changes instead of pushing directly to trunk
+#
+lane :sync_localization do
+  github_token = ENV['GITHUB_TOKEN'] || UI.user_error!('GITHUB_TOKEN environment variable is required')
+
+  sync_branch = 'localization/sync-translations'
+
+  Fastlane::Helper::GitHelper.delete_local_branch_if_exists!(sync_branch)
+  Fastlane::Helper::GitHelper.create_branch(sync_branch)
+
+  po_result = generate_source_po_file(commit_changes: true, skip_date_only_changes: true)
+  translations_result = download_translations(commit_changes: true)
+
+  # Check if either operation resulted in changes
+  has_changes = !po_result.nil? || !translations_result.nil?
+  unless has_changes
+    UI.important('⚠️ No changes to commit')
+    next
+  end
+
+  Fastlane::Helper::GitHelper.delete_remote_branch_if_exists!(sync_branch)
+
+  # Push the localization sync branch to remote
+  push_to_git_remote(set_upstream: true, tags: false)
+
+  pr_result = create_pull_request(
+    repo: GITHUB_REPO,
+    title: 'Sync localization files with GlotPress',
+    body: <<~BODY,
+      This PR synchronizes localization files with GlotPress:
+
+      - **Source PO file**: Updated from English Fluent file for GlotPress sync
+      - **Translations**: Downloaded latest translations from GlotPress and converted to Fluent format
+
+      🔄 This is an automated localization sync.
+    BODY
+    labels: 'Localization',
```

Comment:
Should we also add `reviewers:` or `team_reviewers:` to make sure the PR is not missed and the team have notifications to remember to review it?